packages/vulcan-lib/lib/modules/graphql/graphql.js

```diff
@@ -13,9 +13,37 @@ import Vulcan from '../config.js'; // used for global export
 import { disableFragmentWarnings } from 'graphql-tag';
 
 import collectionToGraphQL from './collection';
+import { generateResolversFromSchema } from './resolvers';
+import {
+  mainTypeTemplate,
+  createDataInputTemplate,
+  updateDataInputTemplate,
+} from '../graphql_templates';
+import getSchemaFields from './schemaFields';
 
 disableFragmentWarnings();
 
+/**
+ * Extract relevant collection information and set default values
+ * @param {*} collection
+ */
+const getCollectionInfos = collection => {
+  const collectionName = collection.options.collectionName;
+  const typeName = collection.typeName
+    ? collection.typeName
+    : Utils.camelToSpaces(_initial(collectionName).join('')); // default to posts -> Post
```

Comment:
Initial is not defined here

Reply:
Thanks for the review. Actually, collection.typename is always defined in createCollection (and createCollection throws an error if you don't pass in a typeName), so I'll just remove this piece of code. On the other hand, collectionName is not necesary.